charts/langwatch/templates/app/secrets.yaml

```
@@ -0,0 +1,26 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.autogen.secretNames.app | default (printf "%s-app-secrets" .Release.Name) }}
+  labels:
+    {{- include "langwatch.labels" . | nindent 4 }}
+type: Opaque
+data:
+  {{- $existingSecret := lookup "v1" "Secret" .Release.Namespace (.Values.autogen.secretNames.app | default (printf "%s-app-secrets" .Release.Name)) }}
+  {{- if $existingSecret }}
+  {{- range $key, $value := $existingSecret.data }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)